packages/npm/cspell-ext.json

```diff
@@ -0,0 +1,42 @@
+// cSpell Settings
+{
+    "id": "npm",
+    "name": "NPM",
+    "description": "NPM dictionary for cspell.",
+    // List of dictionary files to add to the global list of dictionaries
+    "dictionaryDefinitions": [
+        {
+            "name": "npm",
+            "file": "./npm.txt.gz",
+            "description": "NPM dictionary for cspell."
+        }
+    ],
+    // Dictionaries to always be used.
+    // Generally left empty
+    "dictionaries": [],
+    // Language Rules to apply to matching files.
+    // Files are matched on `languageId` and `local`
+    "languageSettings": [
+        {
+            // VSCode languageId. i.e. typescript, java, go, cpp, javascript, markdown, latex
+            // * will match against any file type.
+            "languageId": "javascript",
```

Comment:
Not sure if this should be wildcard or an array of other languages like `typescript`

Reply:
It should be an array. Anything that includes javascript, flow, typescript, react, html, etc.
It is patched up in cspell to include supported types, so no need to be too extensive.